src/role-access/containers/allocate-role/choose-role-error/choose-role-error.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { getLabel } from '../../../../work-allocation-2/utils';
+import {
+  AllocateRoleNavigation
+} from '../../../models';
+
+@Component({
+  selector: 'exui-choose-role-error',
+  templateUrl: './choose-role-error.component.html',
+})
+export class ChooseRoleErrorComponent implements OnInit {
+  @Input() public navEvent: AllocateRoleNavigation;
+
+  public roleCategory: string = '';
+  public jurisdiction: string;
```

Comment:
for consistency change similar to line above:

`public jurisdiction: string = ''`

or maybe change both lines as type is implicitly assigned from value:

`public roleCategory = '';`
`public jurisdiction = '';`

Reply:
Will do